service_container.rst

```diff
@@ -1035,20 +1035,25 @@ to them.
 Linting Service Definitions
 ---------------------------
 
-The ``lint:container`` command checks that the arguments injected into services
-match their type declarations. It's useful to run it before deploying your
-application to production (e.g. in your continuous integration server):
+The ``lint:container`` command performs some additional checks to make sure
+the container is properly configured:
+* checks that the arguments injected into services match their type declarations.
+* checks that the interfaces configured as alias are resolving to a compatible
+service.
```

Comment:
```suggestion

 * Ensure the arguments injected into services match their type declaration ;
 * Ensure the interfaces configured as alias are resolving to a compatible
   service.

```

To avoid the triple use of `check` 🙂

Reply:
Hey 👋 , thanks for the review, fixed (did not apply your suggestion cause ensure is missing an `s` 😛)